exonum-java-binding/core/src/main/java/com/exonum/binding/core/runtime/ServiceRuntime.java

```diff
@@ -386,6 +389,66 @@ public void connectServiceApis(int[] serviceIds, Node node) {
     }
   }
 
+  /**
+   * Get service instance by its name.
+   *
+   * @throws IllegalArgumentException if there is no service with such name
+   */
+  public Service getServiceInstanceByName(String serviceName) {
+    synchronized (lock) {
+      return findService(serviceName)
+          .map(ServiceWrapper::getService)
+          .orElseThrow(() ->
+              new IllegalArgumentException("No service with such name in the Java runtime: "
+                  + serviceName));
+    }
+  }
+
+  /**
+   * Get service instance id by its name.
+   *
+   * @throws IllegalArgumentException if there is no service with such name
+   */
+  public int getServiceIdByName(String serviceName) {
```

Comment:
Is this method really needed? I see it being used (through testkit) in a single test which probably already knows which id it passed when creating the service.